src/main/java/com/uber/jenkins/phabricator/coverage/CoberturaCoverageProvider.java

```diff
@@ -114,7 +116,7 @@ private void computeCoverage() {
         }
 
         // Fallback to scanning for the reports
-        copyCoverageToJenkinsMaster(build);
+        copyCoverageToJenkinsMaster(build, workspace);
```

Comment:
just inline `build.getWorkspace()` here and avoid the local `workspace` variable

Reply:
in the new Jenkins API you can't use Build, but must use Run, but the run interface won't provide a getWorkspace method. I can change this call to this:
```java
copyCoverageToJenkinsMaster(build, getWorkspace());
```
but this is the best I can do.